testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveBucketedTables.java

```diff
@@ -332,6 +444,19 @@ private String hiveTableProperties(BucketingType bucketingType)
         return "TBLPROPERTIES(" + join(",", tableProperties.build()) + ")";
     }
 
+    private String getBucketVersion(BucketingType bucketingType)
+    {
+        switch (bucketingType) {
+            case BUCKETED_DEFAULT:
+            case BUCKETED_V1:
+                return "1";
```

Comment:
Default is 1 or 2, depends on Hive version.

Also change the method name, otherwise someone could use it when _defining_ a table, and then default should result in unset:

```
private String getExpectedBucketVersion(BucketingType bucketingType)
{
    switch (bucketingType) {
        case BUCKETED_DEFAULT:
            return getHiveVersionMajor() < 3 ? "1" : "2";
        case BUCKETED_V1:
            return "1";
        case BUCKETED_V2:
            return "2";
        default:
            throw new UnsupportedOperationException("Not supported for " + bucketingType);
    }
}
```